### ISSUE TITLE
keras Mobilenetv2 convert to TF/CNTK not support ReLU

### ISSUE THREAD

**Issue by ywchiang0819121** [Microsoft/MMdnn#554]
Platform (like ubuntu 16.04/win10): Win 10

Python version:3.6

Source framework with version (like Tensorflow 1.4.1 with GPU):Keras  2.2.4

Destination framework with version (like CNTK 2.3 with GPU):CNTK 2.6 with GPU

Pre-trained model path (webpath or webdisk path):
[https://drive.google.com/file/d/1xt6gHkFdt8mePFNZsEu6bq5mdSz6O-Lj/view?usp=sharing](https://drive.google.com/file/d/1xt6gHkFdt8mePFNZsEu6bq5mdSz6O-Lj/view?usp=sharing)

Running scripts:

mmconvert -sf keras -iw MOBT-128-GRAY-acc-0.9792-loss-0.0502.h5 -df tensorflow -om MOBT-128.pb
&
mmconvert -sf keras -iw MOBT-128-GRAY-acc-0.9792-loss-0.0502.h5 -df cntk -om MOBT-128.dnn

-  Part of Log
KerasParser has not supported operator [ReLU].
{'name': 'Conv1_relu', 'trainable': True, 'max_value': array(6., dtype=float32), 'negative_slope': array(0., dtype=float32), 'threshold': array(0., dtype=float32)}
KerasParser has not supported operator [ReLU].

**Comment by rainLiuplus** [comment]
Hi @ywchiang0819121 , please use the latest code to have a try. Thanks!
